### DISCUSSION TITLE
telegraf meldet Fehler "... strconv.ParseFloat: pasing ... ) bei openDTU mit HM1500

### DISCUSSION THREAD

**Discussion by HRAsoft** [tbnobody/OpenDTU#1247] [General]
Hallo Zusammen,

wenn ich in meiner **telegraf.conf** neben dem **Wechselrichter** auch die Werte der **Solar-Module** 1 bis 4 auslesen möchte, bekomme ich den o.g. Fehler mit den Werten und dem Hinweis "Modul-1": invalid syntax. 
Lese ich nur den Wechselrichter: "solar/+/0/+" aus, läuft alles gut und ich bekomme Daten in der influxDB. Ergänze ich das erste Modul durch den nächsten Eintrag "solar/+/1/+" gibt es die o.g. Fehlermeldung mit "Modul-1".

Wie muss man die Konfiguration anpassen, damit die Solar-Module eigenen Messwerte in die influxDB schreiben und man keinen Fehler bekommt?

Kann mir da jemand einen Tipp geben ?
Vielen Dank im Voraus 

VG Holger

**Comment by HRAsoft** [comment]
Hier noch zwei ScreenShots, die die o.g. Frage deutlicher machen sollen:

**Bild 1 - MQTT-Explorer:**
![Bildschirmfoto vom 2023-08-15 17-54-33](https://github.com/tbnobody/OpenDTU/assets/81297920/4eb56481-6650-44d6-9bad-6c7c910b28e1)

**NBild 2 - telegraf.config:**

Die ausgeklammerten topics hätte ich auch gerne in der influxDB.

![6bxkddAJNPa994V0](https://github.com/tbnobody/OpenDTU/assets/81297920/00865aef-4779-48a6-9ffc-7918026865e9)

**Reply by spueli99** [reply]
Hi,
zum Abschluss der 1.Zeile "solar/+/0/+",
das Komma anfügen.
Alle 4 nachfolgenden Zeilen auskommentieren (die letzte hat dann wieder kein Komma am Ende.
Sollte dann funktionieren.

```
 topics = [
    "solar/+/0/+",
    "solar/+/1/+",
    "solar/+/2/+",
    "solar/+/3/+",
    "solar/+/4/+"
  ]
```
